odiglet/pkg/kube/manager.go

```diff
@@ -38,19 +37,28 @@ func init() {
 func CreateManager() (ctrl.Manager, error) {
 	log.Logger.V(0).Info("Starting reconcileres for runtime details")
 	ctrl.SetLogger(log.Logger)
+
+	odigosNs := env.Current.Namespace
+	nsSelector := client.InNamespace(odigosNs).AsSelector()
+	nameSelector := fields.OneTermEqualSelector("metadata.name", consts.OdigosConfigurationName)
+	odigosConfigSelector := fields.AndSelectors(nsSelector, nameSelector)
+	currentNodeSelector := fields.OneTermEqualSelector("spec.nodeName", env.Current.NodeName)
+
 	return manager.New(config.GetConfigOrDie(), manager.Options{
 		Scheme: scheme,
 		Cache: cache.Options{
 			// ManagedFields are removed to save space. This can save a lot of space and recommended in the cache package.
 			// running `kubectl get .... --show-managed-fields` will show the managed fields.
 			DefaultTransform: cache.TransformStripManagedFields(),
 			ByObject: map[client.Object]cache.ByObject{
+				&corev1.ConfigMap{}: {
+					Field: odigosConfigSelector,
+				},
 				&corev1.Pod{}: {
-					// only watch and list pods in the current node
-					Field: fields.OneTermEqualSelector("spec.nodeName", env.Current.NodeName),
+					Field: currentNodeSelector,
 				},
-				&corev1.Namespace{}: {
```

Comment:
note: this was not used and is leftover from the time we were calculating if workload is instrumented in odiglet